rotkehlchen/tests/exchanges/test_bitpanda.py

```diff
@@ -277,5 +282,28 @@ def mock_bitpanda_query(url: str, **kwargs):  # pylint: disable=unused-argument
         amount=FVal('50'),
         fee_asset=A_EUR,
         fee=FVal('0.01'),
-        link='movementid2')]
+        link='movementid2',
+    ), AssetMovement(
+        location=Location.BITPANDA,
+        category=AssetMovementCategory.DEPOSIT,
+        address='XXX',
```

Comment:
The `XXX` and `REDACTED` was to obfuscate the user's data. Just use some random addresses and transaction hashes instead. Otherwise when we look at it in the future we may wonder wtf this is.